src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+$primary-text-color: #060b35;
+$secondary-text-color: #616070;
+
+.card {
+  width: 198px;
+  height: 406px;
+  padding: 16px;
+  background-color: #fff;
+  box-sizing: border-box;
+  border-radius: 5px;
+  transition: 300ms;
+
+  &__imageframe {
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 40px;
+  }
+
+  &__image {
+    width: 100%;
+  }
+
+  &__title {
+    height: 36px;
+    display: block;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $primary-text-color;
+    margin-bottom: 4px;
+  }
+
+  &:hover {
+    transform: scale(120%);
+
+    & .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $secondary-text-color;
+    text-align: left;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    height: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &__price {
+    height: 18px;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-name {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: $secondary-text-color;
+  }
+
+  &__price-amount {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
```

Comment:
The line-height is set to 18px, which is the same as the font size. Consider adjusting the line-height to ensure proper vertical alignment and readability, especially if the text might wrap or if there are other design considerations.